pandas/tests/test_internals.py

```diff
@@ -297,6 +297,28 @@ def test_split_block_at(self):
         # bs = list(bblock.split_block_at('f'))
         # self.assertEqual(len(bs), 0)
 
+    def test_validate_bool_args(self):
+        invalid_values = [1, "True", [1,2,3], 5.0]
```

Comment:
internals shouldn't have tests for this at all (or even check)

you can assert if u want